frontend/model/state.js

```diff
@@ -64,6 +64,25 @@ sbp('sbp/selectors/register', {
     // if (!state.notifications) {
     //   state.notifications = []
     // }
+
+    // TODO: need to remove the whole content after we release 0.2.*
+    for (const chatRoomId in state.chatRoomUnread) {
+      if (!state.chatRoomUnread[chatRoomId].messages) {
+        state.chatRoomUnread[chatRoomId].messages = []
+      }
+      if (state.chatRoomUnread[chatRoomId].mentions) {
+        state.chatRoomUnread[chatRoomId].mentions.forEach(m => {
+          state.chatRoomUnread[chatRoomId].messages.push(Object.assign({ type: MESSAGE_TYPES.TEXT }, m))
+        })
+        // Vue.delete(state.chatRoomUnread[chatRoomId], 'mentions')
+      }
+      if (state.chatRoomUnread[chatRoomId].others) {
+        state.chatRoomUnread[chatRoomId].others.forEach(o => {
+          state.chatRoomUnread[chatRoomId].messages.push(Object.assign({ type: MESSAGE_TYPES.INTERACTIVE }, o))
+        })
+        // Vue.delete(state.chatRoomUnread[chatRoomId], 'others')
```

Comment:
These two `// Vue.delete(state.chatRoomUnread[chatRoomId], 'others')` lines can be uncommented.